src/main/java/com/stemcraft/SMUtil.java

```diff
@@ -149,7 +149,7 @@ public static boolean loadChunkRadius(Location pos, int radius, boolean force) {
                         if(success) {
                             success = false;
                         }
-                        System.out.println("Failed to load and activate Chunk at X: " + (chunkX * 16) + " Z: " + (chunkZ * 16) + " in " + world.getName());
+                        STEMCraft.getInstance().getLogger().warning("Failed to load and activate Chunk at X: " + (chunkX * 16) + " Z: " + (chunkZ * 16) + " in " + world.getName());
```

Comment:
Let's change this to severe. A chunk failing to load can indicate disk/memory issues and lead to other problems down the track